conformance/base/manifests.yaml

```diff
@@ -201,6 +201,68 @@ spec:
             cpu: 10m
 ---
 apiVersion: v1
+kind: Service
+metadata:
+  name: infra-backend-v4
+  namespace: gateway-conformance-infra
+spec:
+  selector:
+    app: infra-backend-v4
+  ports:
+    - protocol: TCP
+      port: 443
+      targetPort: 8443
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: infra-backend-v4
+  namespace: gateway-conformance-infra
+  labels:
+    app: infra-backend-v4
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: infra-backend-v4
+  template:
+    metadata:
+      labels:
+        app: infra-backend-v4
+    spec:
+      containers:
+      - name: infra-backend-v4
```

Comment:
Do we need a 4th backend (with 2 replicas)? Or can we just add TLS to the existing ones?

Reply:
To conveniently use the existing `echoserver` image, I created a new backend with the environment variables `TLS_SERVER_CERT` and `TLS_SERVER_PRIVKEY` for this image to serve certificates.

Reply:
Please let me know if this is ok.

Reply:
I think this could likely be added to an existing backend without breaking any conformance tests. Would be worth trying at least.

Reply:
The TLS backend, infra-backend-v4, is the only one using port/targetPort 443/8443.  It is also the only one setup with env variables TLS_SERVER_CERT and TLS_SERVER_PRIVKEY, which as mentioned [directs it to validate certs in the echoserver app](https://github.com/kubernetes-sigs/ingress-controller-conformance/blob/master/images/echoserver/echoserver.go#L109.).  

It works fine with 1 replica for now, and will be reused for the rest of the TLSRoute Passthrough conformance tests.   I think putting all of this together, that TLSRoute Passthrough should have its own configuration with non-standard ports and env variables.

I will rename it from infra-backend-v4 to tls-backend so that it is clear that this is a specially configured backend.

Reply:
I'm OK with a new backend here, but can we add this TLS listener to an existing Gateway instead of adding a new Gateway just for this test?